src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/LoongArch64/LA64Base.PlatformNotSupported.cs

```diff
@@ -0,0 +1,217 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma warning disable IDE0060 // unused parameters
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.LoongArch64
+{
+    /// <summary>
+    /// This class provides access to the LA64 base hardware instructions via intrinsics
+    /// </summary>
+    [CLSCompliant(false)]
+#if SYSTEM_PRIVATE_CORELIB
+    public
+#else
+    internal
+#endif
+    abstract class LA64Base
```

Comment:
The name `LA64Base` is about some instructions within the LoongArch64's general ISA.